src/server/game/Scripting/ScriptDefines/GlobalScript.cpp

```diff
@@ -176,3 +176,11 @@ bool ScriptMgr::OnAllowedForPlayerLootCheck(Player const* player, ObjectGuid sou
 
     return true;
 }
+
+void ScriptMgr::OnInstanceIdRemoved(uint32 instanceId)
```

Comment:
```suggestion
/******************************************************************
* Called when an instance Id is deleted, usually because it expired or no
* players are bound to it anymore.
*
*@param instanceId The unique id of the instance
******************************************************************/
void ScriptMgr::OnInstanceIdRemoved(uint32 instanceId)
```

Doxygen

Reply:
Use
```
/**
 * @brief Called when an instance Id is deleted, usually because it expired or no players are bound to it anymore.
 *
 * @param instanceId The unique id of the instance
*/
```
To keep consistency with current docs

Reply:
Adjusted.
Is the hook name ok? I remember we're supposed to use before/after in the names.

Reply:
I am fine with it. As it happens as the instance is removed